### DISCUSSION TITLE
rnaseq-cdr3 preset

### DISCUSSION THREAD

**Discussion by seunghun23** [milaboratory/mixcr#1296] [Q&A] [answered]
Hi, I was trying to run rnaseq-cdr3 preset by `mixcr analyze rnaseq-cdr3 --species hsa` but it returned an error message saying
"No preset with name "rnaseq-cdr3." When I looked at the full list of built-in presets by running `mixcr listPresets`, I realized that there is rna-seq preset, but not rnaseq-cdr3 preset in version 4.4.2. 
Is rnaseq-cdr3 preset no longer supported in this version? 
If so, is there a way to run MixCR on bulk-RNA seq data using only CDR3? 
It looks like rna-seq preset is for full-length

**Comment by mizraelson** [comment, marked as answer]
Hi,
Currently `rna-seq` preset will assemble the longest possible contig for every clone. If only `CDR3` sequence is covered you will still get it in the output. It actually works just as `rnaseq-cdr3` used to.

**Reply by seunghun23** [reply]
Thank you so much for the quick response. I will go ahead and use the rna-seq preset.
